beanstalk_worker/worker_dockerfile_lint.py

```diff
@@ -39,9 +39,10 @@ def lint_job_data(job_data):
     namespace = job_data.get('name_space')
 
     dockerfile = job_data.get("dockerfile")
+    to_email = job_data.get("to_email")
```

Comment:
No field through out the code now uses `to_email`, `notify_email` is the key you should use and access.